nat/traversal/config/parser.go

```diff
@@ -43,5 +43,9 @@ func (c *ConsumerConfigParser) Parse(config json.RawMessage) (ip string, port in
 	if err != nil {
 		return "", 0, "", errors.Wrap(err, "parsing consumer address:port failed")
 	}
-	return cfg.IP, cfg.Port, openvpn.ServiceType, nil
+
+	if cfg.IP == nil {
```

Comment:
Shouldn't we keep other parameters if only IP is empty? Or maybe return an error if it's a mandatory argument?

Reply:
And another thing that confuses me, why we are using a pointer to string here? Isn't it enough to compare with empty string?
I'd use `net.IP` for every place where it's possible as a best case.

Reply:
> Or maybe return an error if it's a mandatory argument?

Was thinking the same - it doesn't make much sense to return `nil` error when method failed to execute it's duty.

Reply:
Parsing fails if there is no IP inside message. Just wanted to make it optional and don't get a parsing error.